charter.html

```diff
@@ -268,10 +268,14 @@ <h2>Success Criteria</h2>
     specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
 
           <p>Each normative specification should contain separate sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+          
+          <p>Any proposed feature should reduce the data available about a user.</p>
```

Comment:
Does this belong in this PR? It seems unconnected. Additionally: presumably this isn't _strictly_ reduce (ie features that don't change the amount of data available about a user are fine)?

Reply:
```suggestion
          <p>Any proposed feature should increase the protection of data about a user.</p>
```

Reply:
```suggestion
          <p>Any proposed feature should increase the protection of data about a user.</p>
```

Reply:
@jaylett-annalect The goal here is to provide an outline for exactly what the risk involved is (users' data being emitted) for the question of utility and safeguards below.  

> features that don't change the amount of data available about a user are fine

The argument I would make is that while those features that maintain the status quo of data flow between the user and site might or might not be fine (depending on the feature), those features that do not explicitly reduce the flow of user data, or give a user the ability to choose to reduce the flow of data don't really belong in this group's workflow, since they wouldn't be accomplishing the Privacy part of Private Ad Technology. This specific separation is what I'm hoping will be clear here. We don't want to create an overly broad scope for the group.

Reply:
I may be being overly pedantic, but something like IPA is not changing the volume of user data flowing around. It's introducing data that would not be considered user data (assuming the privacy guarantees hold up!). But it by itself does nothing to _reduce_ flows; it relies on (for instance) third-party cookie deprecation to achieve that (which is already in place on some browsers, so not reducing the flow of data there either). Or I suppose it could rely on deprecation of current attribution techniques, inclusive of those dependent on 3PC. So perhaps it could be "any proposed feature should reduce the data available about a user compared to current techniques and practice" ... although that's a bit fluffy perhaps.

I understand and support the goal here, I'm just not sure the text is right.